src/openssl/hmac.cr

```diff
@@ -12,28 +13,8 @@ class OpenSSL::HMAC
   # Returns the HMAC digest of *data* using the secret *key*.
   #
   # It may contain non-ASCII bytes, including NUL bytes.
-  #
-  # *algorithm* is a `Symbol` of a supported digest algorithm:
```

Comment:
Please leave a description of the `algorithm` argument. Obviously we don't need a list of values when there is a reference to `OpenSSL::Algorithm`.

Reply:
As I'm not a very good writer in english (not my native language), do you have a suggestion of phrasing?
How about 
```
# *algorithm* is a `OpenSSL::Algorithm`.
```
or 
```
# *algorithm* represents the digest algorithm as `OpenSSL::Algorithm`.
```

Reply:
How about: ``*algorithm* specifies which `OpenSSL::Algorithm` is to be used.``

Reply:
Nice! I'll use that